runtime/testing/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ kotlin {
             dependencies {
                 implementation("org.jetbrains.kotlin:kotlin-test:$kotlinVersion")
                 api("org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutinesVersion")
+                api(project(":runtime:utils"))
```

Comment:
question

Why does this need `utils`?

Reply:
See my comment on the `aws-sdk-kotlin` PR. It actually doesn't use the annotation, it's because we opt-in for the [entire runtime](https://github.com/awslabs/smithy-kotlin/blob/main/runtime/build.gradle.kts#L16)